src/Hydration.js

```diff
@@ -0,0 +1,11 @@
+class Hydration {
+  constructor (hydrationData) {
+    this.id = hydrationData.userID;
+    this.date = hydrationData.date;
+    this.numOunces = hydrationData.numOunces;
+  }
+}
+
+if (typeof module !== 'undefined') {
+  module.exports = Hydration;
+}
```

Comment:
If Hydration class ends up not being used in scripts.js, let's make sure to remember to remove it at the end of the project